_posts/en/newsletters/2020-10-14-newsletter.md

```diff
@@ -150,7 +150,13 @@ FIXME
 [Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
 (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
 
-- [Bitcoin Core #19954][] Complete the BIP155 implementation and upgrade to TORv3  FIXME:adamjonas
+- [Bitcoin Core #19954][] completes the [BIP155][] implementation, also referred
+  to as [addr v2][topic addr v2]. As previously covered in
+  [Newsletter #110][news110 addrv2], this culminates the upgrade to support
+  TOR v3, I2P, and other networks with longer endpoint addresses that do not
```

Comment:
Might want to rephrase this to make clear that only Tor v3 is currently supported; I2P and other networks may be added in the future.

Reply:
Done